docs/gtk/install.md

```diff
@@ -13,38 +13,66 @@ The simplest case                      {#gtk_simple}
 If you compile wxWidgets on Linux for the first time and don't like to read
 install instructions just do the following in wxWidgets directory:
 
-    > mkdir buildgtk
-    > cd buildgtk
-    > ../configure --with-gtk
-    > make
-    > sudo make install
-    > sudo ldconfig
+    $ mkdir buildgtk
+    $ cd buildgtk
+    $ ../configure --with-gtk
+    $ make
+    $ sudo make install
+    $ sudo ldconfig
 
-[if you get "ldconfig: command not found", try using "/sbin/ldconfig"]
+(if you get "ldconfig: command not found", try using `/sbin/ldconfig`)
 
-If you don't do the 'make install' part, you can still use the libraries from
-the buildgtk directory, but they may not be available to other users.
-
-If you want to remove wxWidgets on Unix you can do this:
-
-    > sudo make uninstall
-    > sudo ldconfig
+If you don't do the `make install` part, you can still use the libraries from
+the `buildgtk` directory, but they may not be available to other users.
 
 Note that by default, GTK 3 is used. GTK 2 can be specified
-with --with-gtk=2.
+with `--with-gtk=2` configure option.
 
-If you use CMake, please see @ref overview_cmake for
+If you want to use CMake, please see @ref overview_cmake for
 building wxWidgets using it.
 
 
+Prerequisites                      {#gtk_prereq}
+-------------
+
+To build wxWidgets, you need to have a number of libraries on your system.
+While all of them may be compiled from source, under Linux systems it is much
+more common to install the development packages containing the headers and
+library files.
+
+At the very least, you will need GTK libraries themselves. Unfortunately the
+exact package names differ between different Linux distributions, but for
+Debian and Debian-based distribution these libraries are part of `libgtk-3-dev`
+package, while in Fedora and other RPM-based distributions the same package is
+known as `gtk3-devel`.
+
+For OpenGL support, you need `libgl1-mesa-dev` and `libglu1-mesa-dev` packages
+under Debian and `mesa-libGL-devel` and `mesa-libGLU-deve` under Fedora. For
+EGL support, `libegl1-mesa-dev` or `mesa-libEGL-devel` is needed.
+
+wxMediaCtrl implementation requires GStreamer, i.e. either
+`libgstreamerVERSION-dev` or `gstream-VERSION-devel`, where `VERSION` can be
+either 0.10 or 1.0 (recommended).
```

Comment:
On Fedora, the package name is `gstreamer1-devel` but for some reason we install `gstreamer1-plugins-base-devel`.  Can't remember the reasoning for the plugins-base package.  Similarly, on Debian we install `libgstreamer-plugins-base1.0-dev`.

Reply:
Thanks for noticing this, we indeed need `libgstreamer-plugins-base1.0-dev` as some headers we use are contained in it, and not `libgstreamer1.0-dev` itself (which is its dependency).